facturacion_mexico/install.py

```
@@ -23,6 +23,37 @@ def after_install():
+	except Exception as e:
+		frappe.log_error(frappe.get_traceback(), "[FMX][Install] Error creating default payment terms")
+		frappe.logger().warning(f"⚠️ No se pudieron crear Payment Terms FM: {e}")
+
+	# Item Groups de gasto CFDI Recibidos
+	try:
+		from facturacion_mexico.setup.cfdi_received_expense_item_groups import (
+			ensure_cfdi_received_expense_item_groups,
+		)
+
+		ensure_cfdi_received_expense_item_groups()
+	except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)